core/templates/dev/head/appSpec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
Hi @ankita240796 -- this is still showing up as a separate add/delete. Is it possible to make it look like a rename instead?

Reply:
I tried doing a rename but it works up to some commits and when I further push more commits, rename is not detected. I found a similar issue [here](https://github.com/sindresorhus/refined-github/pull/1750#issuecomment-460886960). I was not able to find a solution for this but to make the review easier, I made separate commits for each rename operation and only the first two commits in this PR need a review.